open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/activedata/ActiveDataPanel.java

```diff
@@ -616,6 +617,9 @@ protected Component getViewByComboBox()
         viewBy.addActionListener(e ->
         {
             String activeType = viewBy.getSelectedItem().toString();
+            QuantifyToolboxUtils.collectMetric(myToolbox,
+                    "mist3d.layer-manager.group-by." + activeType.toLowerCase().replaceAll(" ", "-"));
```

Comment:
Since `String.toLowerCase().replaceAll(" ", "-")` will likely be used a lot, have you considered either making a utility method that does that, or do that in collectMetric() et al to all strings passed in so that the caller doesn't always have to do that?

Reply:
yea, that'd be a good idea. I'll do that.